pkg/vault/contracts/Router.sol

```diff
@@ -671,10 +670,10 @@ contract Router is IRouter, RouterCommon, ReentrancyGuardTransient {
      * @param wrappedToken Address of the wrapped token that implements IERC4626
      * @param amountUnderlyingRaw Amount of underlying tokens that will be deposited into the buffer
      * @param amountWrappedRaw Amount of wrapped tokens that will be deposited into the buffer
-     * @param sharesOwner Address of contract that will own the deposited liquidity. Only
-     *        this contract will be able to remove liquidity from the buffer
-     * @return issuedShares the amount of tokens sharesOwner has in the buffer, expressed in underlying token amounts
-     *         (it is the BPT of vault's internal linear pools)
+     * @param sharesOwner Address of contract that will own the deposited liquidity. Only this contract will be able
+     * to remove liquidity from the buffer
+     * @return issuedShares the amount of tokens sharesOwner has in the buffer, expressed in underlying token amounts.
+     * (This is the BPT of an internal ERC4626 token buffer)
```

Comment:
```suggestion
     * (This is the BPT of an internal ERC4626 buffer)
```
We called ERC4626 buffer before, so removing the token word to keep the standard (or put "token" in the suggestion above)

Reply:
Ok; I think we'll say "ERC4626 buffer" and "yield-bearing token buffer," since "yield-bearing token" is the meaningful phrase here, and ERC4626 already implies token.